docs/usage.md

```diff
@@ -6,14 +6,18 @@
 
 ## Introduction
 
-<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
+General usage:
+
+```bash
+nextflow run nf-core/pgdb -profile <docker/singularity/podman/conda/institute> --ensembl_name homo_sapiens
+```
 
 ## Running the pipeline
 
 The typical command for running the pipeline is as follows:
 
 ```bash
-nextflow run nf-core/pgdb --input '*_R{1,2}.fastq.gz' -profile docker
+nextflow run nf-core/pgdb --taxonomy 9606 --altorfs true -profile docker
```

Comment:
CLI param flags are boolean by default if no argument is given, so `true` is not needed.

```suggestion
nextflow run nf-core/pgdb --taxonomy 9606 --altorfs -profile docker
```